test/test_aioredis2.py

```diff
@@ -0,0 +1,255 @@
+import asyncio
+import distutils.version
+import re
+
+import pytest
+import aioredis
+import async_timeout
+from async_generator import yield_, async_generator
+
+import fakeredis.aioredis
+
+
+aioredis2 = aioredis.__version__ >= distutils.version.StrictVersion('2.0.0a1')
+pytestmark = [
+    pytest.mark.asyncio,
+    pytest.mark.skipif(not aioredis2, reason="Test is only applicable to aioredis 2.x")
```

Comment:
The unit tests in `test_aioredis2.py` are presumably meant for aioredis 2.x, so I think the original sense is correct.

Reply:
A test is being skipped here if it's not marked as being for aioredis 2.x so I think the reason should be what I've proposed? Or am I missing something?

Reply:
The way I read it is that `not aioredis2` means that aioredis 1 is installed. Then all tests in `test_aioredis2` will be skipped, because they are meant to be run with aioredis 2.x.

Reply:
Ah, yeah. Sorry. 😅

Reply:
I think the confusion on my side came from the fact that I thought that in the 1.x tests the message was also talking about 2.x

Reply:
No worries - I first had to figure out that `pytestmark` actually does something... Ah, the magic of pytest 😂